### DISCUSSION TITLE
Traffic Graphs Y-Axis Labels

### DISCUSSION THREAD

**Discussion by adlai** [community/community#37565] [Repositories]
I will not provide a screenshot, to avoid quibbles about light and dark themes; the following explanation is probably not complicated enough to warrant one, anyway.

When the traffic graphs display values in sufficiently small ranges, the Y-axis labelling includes decimals. Unless someday Insights will account for occurrences such as incomplete clones, the axis labels should only be non-negative integers, to reflect the possible data values.

**Comment by adlai** [comment]
I categorized the suggestion under "Repositories", and will not chase
the volunteer complaining about insufficiently accurate information
any further tonight.

Happy Hallowe'en!

**Comment by ossanna16** [comment]
Hi there @adlai! Thank you for your question. Screenshots would actually be really helpful in this case!

**Reply by adlai** [reply]
> Screenshots would actually be really helpful in this case!

Seriously... what language does the UX team read?

**Reply by ossanna16** [reply]
Hi @adlai! In order for us to help you, it would be great if you submitted more details, including screenshots, regarding your query. If you are unable to do so, that is fine and I understand, but then it will be hard for us to help you resolve your issue.

**Reply by adlai** [reply]
First, to test the display of images...

![screw](https://user-images.githubusercontent.com/94171/199652617-7a439c13-8a70-48d2-a301-507597889479.png)
![scrum](https://user-images.githubusercontent.com/94171/199652424-6c5ea481-4183-47f5-b0fe-d1109a6b0c0b.png)
![scrat](https://user-images.githubusercontent.com/94171/199652642-e56724c7-34d8-4a24-8c53-05b3ab2d4f0a.png)

Now, what sort of information belongs in the screenshot, and what does not? The sad thing about doing this for free is that I could do this all day, and night, and only stop for flipping on dark mode.

**Reply by adlai** [reply]
> Hi @adlai! In order for us to help you, it would be great if you submitted more details, including _screenshots_, regarding your query. If you are unable to do so, that is fine and I understand, but then it will be hard for us to help you resolve your issue.

Is one enough? It will take me a long time to fake a mockup, and your coworkers can probably do it better than I.

**Reply by adlai** [reply]
> [...] if you submitted more details, including screenshots, regarding your query. [...]

Here's one attempt, with the data range significantly longer than only zero and one:

![traffic graph with supra-natural axes](https://user-images.githubusercontent.com/94171/213518095-602a1c60-59a7-4f42-a1bf-cc2a0bee1dc9.png)

I hope that the wording in the image's alt-text does not give the wrong impression; there is nothing arguably incorrect about the labelling, although at minimum, somebody chasing perpetual optimisation could argue that the marks of 0.5 and 1.5 can be deleted from the upper graph.

**Reply by adlai** [reply]
If _any_-~some~one _else_ is interested in presenting useful information beyond the granularity currently available in that graph, it is probably more appropriate to take those suggestions to site-wide discussions about what information the Traffic page provides, similar to the conversations already opened about Contributors.

**Comment by adlai** [comment]
> Hi @adlai! In order for us to help you, [...]

THIS IS NOT VIRTUAL REALITY, IT'S ONE PAGE ON ONE WEB SITE. DO NOT GREET ME UNLESS YOU VISIT THE CUBE FARM WHERE I SUFFER.


> it would be great if you submitted more details, including screenshots, [...]

How many mockups will y'all be milking out the volunteers? @ossanna16
